lib/error.js

```diff
@@ -39,6 +39,12 @@ var ERRNO = {
   BOUNCE_COMPLAINT: 133,
   BOUNCE_HARD: 134,
   BOUNCE_SOFT: 135,
+  EMAIL_EXISTS: 136,
+  EMAIL_DELETE_PRIMARY: 137,
+  SESSION_UNVERIFIED: 138,
+  USER_PRIMARY_EMAIL_EXISTS: 139,
+  VERIFIED_PRIMARY_EMAIL_EXISTS: 140,
+  UNVERIFIED_PRIMARY_EMAIL_NEWLY_CREATED: 141,
```

Comment:
Could you explain for this is with a doc comment or we might need to rename this...